SimonImanigirimpuhwe/my-brand-website
# #2 ft(landing-page): my brand website landing page

#### What does this PR do?

- Add my brand website landing page

#### Description of Task to be completed?

- Add profile picture

- Add navigation links to the page

- Add social media links

#### How should this be manually tested?

- Navigate to the page you will be able to see profile picture, navigation links, social media links connected to the actual account.

#### Any background context you want to provide?

- N/A

#### What are the relevant Trello stories?

- [#2](https://trello.com/c/jfMeexOv/2-landing-page)

#### Screenshots (if appropriate)
![landing-page-screenshoot](https://user-images.githubusercontent.com/54619678/93466502-a1dff200-f8ec-11ea-9f9a-2ad64f9b279a.png)


#### Questions:

- N/A
